rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml

```diff
@@ -15,7 +15,7 @@ when a process is started whose name or code signature resembles commonly abused
 indicating the host has not seen this RAT process started before within the last 30 days.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
```

Comment:
I'm dropping Winlogbeat and sysmon here, as it doesn't support either code signature information or caseless fields. Adding endgame as it can match the code signature information although it doesn't have a caseless field.

Reply:
can't we add `?` for optional fields and keep the process.name part ? or duplicate the rule for sysmon/winlogbeat using process.name only ?

Reply:
Optional fields only work in EQL
A duplicate could do the job halfway because we would have the risk of simply not getting matches because of the case

Reply:
sysmon IIRC captures original file name, it can be used no ?

Reply:
Or duplicate the `process.name.caseless` list to `process.name` for now

Reply:
https://github.com/elastic/kibana/pull/179821